docs/tutorial-2/multi-node.rst

```diff
@@ -0,0 +1,189 @@
+
+   eyJuYW1lIjoibm9kZS0zIiwic2VjcmV0IjoiYTZjYWJjOTZiNDJkYjg0YTRkZTFiY2MzY2VkYTI1M2Y4MTU1ZTNhYjAwYWUyOWY1MDA4ZWQzY2RmOTYzMjBmMiIsImZpbmdlcnByaW50IjoiMmU0MmEzYjEwYTg1MDcwYTQ1MDcyODQxZjAyNWY5NGE0OTc4NWU5MGViMzZmZGY0ZDRmODhhOGQyYjQ0MmUyMyIsImpvaW5fYWRkcmVzc2VzIjpbIjEwLjI0Ni4xMTQuMTE6NzQ0MyJdfQ==
+
+Keep these tokens in a safe place. They'll be needed in the next step.
+
+Join the non-primary nodes to the cluster
+-----------------------------------------
+
+The :command:`cluster join` command is used to join nodes to a cluster.
+
+On **node-2**, add the machine to the cluster using the token assigned to
+node-2:
+
+.. code-block:: none
+
+   sudo microceph cluster join eyJuYW1lIjoibm9kZS0yIiwic2VjcmV0IjoiYmRjMzZlOWJmNmIzNzhiYzMwY2ZjOWVmMzRjNDM5YzNlZTMzMTlmZDIyZjkxNmJhMTI1MzVkZmZiMjA2MTdhNCIsImZpbmdlcnByaW50IjoiMmU0MmEzYjEwYTg1MDcwYTQ1MDcyODQxZjAyNWY5NGE0OTc4NWU5MGViMzZmZGY0ZDRmODhhOGQyYjQ0MmUyMyIsImpvaW5fYWRkcmVzc2VzIjpbIjEwLjI0Ni4xMTQuMTE6NzQ0MyJdfQ==
+
+On **node-3**, add the machine to the cluster using the token assigned to
+node-3:
+
+.. code-block:: none
+
+   sudo microceph cluster join eyJuYW1lIjoibm9kZS0zIiwic2VjcmV0IjoiYTZjYWJjOTZiNDJkYjg0YTRkZTFiY2MzY2VkYTI1M2Y4MTU1ZTNhYjAwYWUyOWY1MDA4ZWQzY2RmOTYzMjBmMiIsImZpbmdlcnByaW50IjoiMmU0MmEzYjEwYTg1MDcwYTQ1MDcyODQxZjAyNWY5NGE0OTc4NWU5MGViMzZmZGY0ZDRmODhhOGQyYjQ0MmUyMyIsImpvaW5fYWRkcmVzc2VzIjpbIjEwLjI0Ni4xMTQuMTE6NzQ0MyJdfQ==
+
+Add storage
+-----------
+
+.. warning::
+
+   This step will remove the data found on the target storage disks. Make sure
+   you don't lose data unintentionally.
+
+On **each** of the three machines, use the :command:`disk add` command to add
+storage:
+
+.. code-block:: none
+
+   sudo microceph disk add /dev/vdb --wipe
+
+Adjust the above command per machine according to the storage disks at your
+disposal.
+
+Check MicroCeph status
+----------------------
+
+On any of the three nodes, the :command:`status` command can be invoked to
+check the status of MicroCeph:
+
+.. code-block:: none
+
+   sudo microceph status
+
+   MicroCeph deployment summary:
+   - node-01 (10.246.114.11)
+     Services: mds, mgr, mon, osd
+     Disks: 1
+   - node-02 (10.246.114.47)
+     Services: mds, mgr, mon, osd
+     Disks: 1
+   - node-03 (10.246.115.11)
+     Services: mds, mgr, mon, osd
+     Disks: 1
+
+Machine hostnames are given along with their IP addresses. The MDS, MGR, MON,
+and OSD services are running and each node is supplying a single disk, as
+expected.
+
+Manage the cluster
+------------------
+
+Your Ceph cluster is now deployed and can be managed by following the resources
+found in the :doc:`Howto <../how-to/index>` section.
+
+The cluster can also be managed using native Ceph tooling if snap-level
+commands are not yet available for a desired task by appending a native command
+to the :command:`microceph` command. This is the equivalent to the standard
+:command:`ceph status` command for instance:
+
+.. code-block:: none
+
+   microceph.ceph status
```

Comment:
We have now aliases in for the high-level ceph cli commands -- e.g. `ceph`, `rbd`, `radosgw-admin` all point to `microceph.ceph`, `microceph.rbd` etc. 
So from my perspective we don't need to mention the `microceph.` prefixed commands here

Reply:
Interesting ok!